mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,620 @@
+    def load_cens(self, *args, **kwargs):
+        return load_cens(*args, **kwargs)
+
+    @core.copy_docs(load_crema)
+    def load_crema(self, *args, **kwargs):
+        return load_crema(*args, **kwargs)
+
+    @core.copy_docs(load_hpcp)
+    def load_hpcp(self, *args, **kwargs):
+        return load_hpcp(*args, **kwargs)
+
+    @core.copy_docs(load_key)
+    def load_key(self, *args, **kwargs):
+        return load_key(*args, **kwargs)
+
+    @core.copy_docs(load_mfcc)
+    def load_mfcc(self, *args, **kwargs):
+        return load_mfcc(*args, **kwargs)
+
+    @core.copy_docs(load_madmom)
+    def load_madmom(self, *args, **kwargs):
+        return load_madmom(*args, **kwargs)
+
+    @core.copy_docs(load_tags)
+    def load_tags(self, *args, **kwargs):
+        return load_tags(*args, **kwargs)
+
+    def filter_index(self, search_key):
+        """Load from da_tacos genre dataset the indexes that match with search_key.
+
+        Args:
+            search_key (str): regex to match with folds, mbid or genres
+
+        Returns:
+             dict: {`track_id`: track data}
+
+        """
+        data = {k: v for k, v in self._index["tracks"].items() if search_key in k}
+        return data
+
+    def benchmark_tracks(self):
+        """Load from da_tacos dataset the benchmark subset tracks.
+
+        Returns:
+            dict: {`track_id`: track data}
+
+        """
+        return self.filter_index("benchmark#")
+
+    def coveranalysis_tracks(self):
+        """Load from da_tacos dataset the coveranalysis subset tracks.
+
+        Returns:
+            dict: {`track_id`: track data}
+
+        """
+        return self.filter_index("coveranalysis#")
+
+
+    def download(self, partial_download=None, force_overwrite=False, cleanup=False):
+        """Download the dataset
+        Args:
+            partial_download (str):
+                A list of keys of remotes to partially download.
+                If None, crema data is downloaded
+            force_overwrite (bool):
+                If True, existing files are overwritten by the downloaded files.
+            cleanup (bool):
+                Whether to delete any zip/tar files after extracting.
+        Raises:
+            ValueError: if invalid keys are passed to partial_download
+            IOError: if a downloaded file's checksum is different from expected
+        """
+        if partial_download is None or partial_download == "crema":
+            partial_download = ["benchmark_crema", "coveranalysis_crema"]
+        elif "all" in partial_download:
+            partial_download = ["metadata", "benchmark_cens", "benchmark_crema", "benchmark_hpcp", "benchmark_key",
+                                "benchmark_madmom", "benchmark_mfcc", "coveranalysis_tags", "coveranalysis_cens",
+                                "coveranalysis_crema", "coveranalysis_hpcp", "coveranalysis_key",
+                                "coveranalysis_madmom", "coveranalysis_mfcc"]
```

Comment:
Same for these - if the "all" case is different from the set of all keys in REMOTES, then specify the list in the Index like I commented above.
```suggestion
```